server/src/completion/YmlCompletionItemsProvider.ts

```diff
@@ -62,4 +62,14 @@ export class YmlCompletionItemsProvider {
             })
             .filter((elem) => !!elem);
     }
+
+    /**
+     * Find the *first* completion item that has `entityName` as name.
+     *
+     * @param entityName The name of the entity searched for.
+     * @returns The first completion item that has the same label as `entityName`, `null` otherwise.
```

Comment:
Not quite what's actually happening.

```suggestion
     * @returns The first completion item that has `entityName` as its label, `null` otherwise.
```